internal/server/metrics.go

```diff
@@ -0,0 +1,45 @@
+package server
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/collectors"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	namespace = "perseus"
```

Comment:
Nit: I understand this is metrics-only, but as it's in the `server` package, should we name it something more explicit like `metricsNamespace`?

Reply:
This is actually used in multiple places now. I'll rename it to `serviceName`